teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -310,12 +310,12 @@ interface BuildConfiguration {
     var buildNumberFormat: String
 
     fun runBuild(parameters: Map<String, String>? = null,
-                 queueAtTop: Boolean = false,
-                 cleanSources: Boolean = false,
-                 rebuildAllDependencies: Boolean = false,
+                 queueAtTop: Boolean? = null,
+                 cleanSources: Boolean? = null,
+                 rebuildAllDependencies: Boolean? = null,
                  comment: String? = null,
                  logicalBranchName: String? = null,
-                 personal: Boolean = false): Build
+                 personal: Boolean? = null): Build
```

Comment:
What is the default value for `personal` option? Is it possible to set up a build configuration to run all builds as personal by default?

Reply:
Nope (not at the moment).
That was my thoughts after writing PR. Probably we should leave `queueAtTop`, `personal`, `rebuildAllDependencies` options as is, with `false` default value (they're a part of build run request). Meanwhile `cleanSources` is a part of build configuration.